imports/plugins/core/accounts/server/no-meteor/mutations/inviteShopMember.js

```diff
@@ -0,0 +1,159 @@
+import _, { difference } from "lodash";
+import SimpleSchema from "simpl-schema";
+import Random from "@reactioncommerce/random";
+import ReactionError from "@reactioncommerce/reaction-error";
+import { Meteor } from "meteor/meteor";
+import { Accounts as MeteorAccounts } from "meteor/accounts-base";
+import Reaction from "/imports/plugins/core/core/server/Reaction";
+import getCurrentUserName from "/imports/plugins/core/accounts/server/no-meteor/util/getCurrentUserName";
+import getDataForEmail from "/imports/plugins/core/accounts/server/util/getDataForEmail";
+
+const inputSchema = new SimpleSchema({
+  email: String,
+  groupId: String,
+  name: String,
+  shopId: String
+});
+
+/**
+ * @name accounts/inviteShopMember
+ * @memberof Mutations/Accounts
+ * @summary Invite new admin users (not consumers) to secure access in the dashboard to permissions
+ * as specified in packages/roles
+ * @param {Object} context - GraphQL execution context
+ * @param {Object} input - Necessary input for mutation. See SimpleSchema.
+ * @param {String} input.shopId - shop to invite user
+ * @param {String} input.groupId - groupId to invite user
+ * @param {String} input.email - email of invitee
+ * @param {String} input.name - name of invitee
+ * @return {Promise<Object>} with boolean of found new account === true || false
+ */
+export default async function inviteShopMember(context, input) {
+  inputSchema.validate(input);
+  const { appEvents, collections, user: userFromContext, userHasPermission, userId: userIdFromContext } = context;
+  const { Accounts, Groups, Shops, users } = collections;
+  const {
+    email,
+    groupId,
+    name,
+    shopId
+  } = input;
+
+  if (!userHasPermission(["reaction-accounts"], shopId)) {
```

Comment:
Maybe you should make a more specific `account/invite` for this while you're in here?